ZConfig/info.py

```diff
@@ -64,13 +64,15 @@ class BaseInfo(object):
 
     def __init__(self, name, datatype, minOccurs, maxOccurs, handler,
                  attribute):
-        if maxOccurs is not None:
-            if maxOccurs < 1:
-                raise ZConfig.SchemaError(
-                    "maxOccurs must be at least 1")
-            if minOccurs is not None and minOccurs > maxOccurs:
-                raise ZConfig.SchemaError(
-                    "minOccurs cannot be more than maxOccurs")
+        assert maxOccurs is not None, "Use Unbounded for an upper bound, not None"
+        assert minOccurs is not None, "Use 0 for a lower bound, not None"
```

Comment:
Ugh, I hate to see `assert` statements in non-test code:  `python -O` causes them to be optimized out.